enrollment-server-common/src/main/java/com/wultra/app/enrollmentserver/common/annotation/PublicApi.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.enrollmentserver.common.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+/**
+ * Marker for interfaces intended to be called by downstream projects. Implementation may not be exposed by the core functionality.
+ * <p>
+ * New methods can be added.
+ * Those clients that used to call the previously existing elements, don't need to care about the new ones.
```

Comment:
I am not sure I fully understand this sentence. Is this related to new methods? What are the elements exactly?